tests/unit/utils/Utils.cpp

```diff
@@ -68,3 +70,18 @@ char * mktemp_cst(char * __template)
 
     return __template;
 }
+
+char* generate_file_name(char* file_name)
```

Comment:
@MikeLing deprecated :) true.... ignore these comments :) the naming of the fucntion though should reflect the temp-ness :)
btw the return type should be void since the passed `char*` will actually contain the right filename :)